cpp/formatting/README.md

```diff
@@ -30,8 +30,13 @@ A C++ snippet has the following structure:
 
 // optional *markown* description
 // Multiline description is supported
+// clang-format: KEY: VALUE
+// clang-format: KEY: VALUE
+
 
 template <typename T>
 your cpp(code);
 ```
-If a C++ snippet highlights a particular clang-format configuration key, then it should be named after it.
+
+If a C++ snippet highlights a particular clang-format configuration key, then it should be named after it. Additional Clang-Format key/value
```

Comment:
Really long line. Text should break at 80 chars!

Reply:
It's breaking only code?

Reply:
Agree. we should break just like code. @matz-e would you agree using 100 chars for everything?

Reply:
> Really long line. Text should break at 80 chars!
You are right.
This README.md is not enforced by C++ formatting rules.

Reply:
> Agree. we should break just like code. @matz-e would you agree using 100 chars for everything?

I would prefer to have 80 columns, but 100 is more pragmatic.

Reply:
> > Agree. we should break just like code. @matz-e would you agree using 100 chars for everything?
> 
> I would prefer to have 80 columns, but 100 is more pragmatic.

Same. This is Markdown, so to me that's an editor setting. In code documentation should be wrapped shorted than code, potentially for me. If that's not an option, I can live with that.